cmd/scheduler/options/options.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/pflag"
+
+	"github.com/dapr/dapr/pkg/metrics"
+	"github.com/dapr/dapr/pkg/modes"
+	"github.com/dapr/dapr/pkg/security"
+	securityConsts "github.com/dapr/dapr/pkg/security/consts"
+	"github.com/dapr/kit/logger"
+)
+
+type Options struct {
+	Port                 int
+	HealthzPort          int
+	HealthzListenAddress string
+
+	ListenAddress    string
+	TLSEnabled       bool
+	TrustDomain      string
+	TrustAnchorsFile *string
+	SentryAddress    string
+	PlacementAddress string
+	Mode             string
+
+	ID                      string
+	ReplicaID               uint32
+	ReplicaCount            uint32
+	EtcdInitialPeers        []string
+	EtcdDataDir             string
+	EtcdClientPorts         []string
+	EtcdClientHTTPPorts     []string
+	EtcdSpaceQuota          int64
+	EtcdCompactionMode      string
+	EtcdCompactionRetention string
+
+	Logger  logger.Options
+	Metrics *metrics.Options
+
+	taFile string
+}
+
+func New(origArgs []string) (*Options, error) {
+	// We are using pflag to parse the CLI flags
```

Comment:
This is probably not needed for a new service, since it was only for backwards-compat for daprd.

Reply:
The comment or the `OrigArgs`?

Reply:
The entire block where the args are transformed. That was only needed in daprd, not for control plane services

Reply:
From what I can find, it looks like the other control plane services include this diff as well. [See the PR where you added this to all the other control plane services.](https://github.com/dapr/dapr/commit/ace5b645edc32c1f95f6e689d1f65a939c47153e#diff-b0e58d8b097ed95fbd37b293e3e629e39829972fc09efce2360a15348ea03914) I understand that since Scheduler is a new service its not necessary to maintain backwards compatability, but maybe for consistency sake across the services it should remain- thoughts?

Reply:
I'd remove it. That block is only useful for legacy reasons

Reply:
@cicoyle was this change request addressed?